elements/rh-button/rh-button.css

```diff
@@ -0,0 +1,490 @@
+}
+
+/******************************
+ *                            *
+ *          PRIMARY           *
+ *                            *
+ ******************************/
+
+button:active {
+  --_color: var(--_active-color, var(--_primary-active-color));
+  --_background-color: var(--_active-background-color, var(--_primary-active-background-color));
+  --_border-width: var(var(--_active-border-width), var(--_primary-active-border-width));
+}
+
+button:focus {
+  --_color: var(--_focus-color, var(--_primary-focus-color));
+  --_background-color: var(--_focus-background-color, var(--_primary-focus-background-color));
+  --_border-width: var(--_focus-border-width, var(--_primary-focus-border-width));
+}
+
+button:active,
+button:focus {
+  outline:
+    var(--rh-border-width-md, 2px)
+    solid
+    var(--_focus-outline-color, var(--rh-color-interactive-blue-darker, #0066cc));
+}
+
+button:hover {
+  --_color: var(--_hover-color, var(--_primary-hover-color));
+  --_background-color: var(--_hover-background-color, var(--_primary-hover-background-color));
+  --_border-width: var(--_hover-border-width, var(--_primary-hover-border-width));
+}
+
+/******************************
+ *                            *
+ *         SECONDARY          *
+ *                            *
+ ******************************/
+
+:host([variant="secondary" i]) button {
+  --_default-color: var(--_secondary-color);
+  --_default-background-color: var(--_secondary-background-color);
+  --_default-border-color: var(--_secondary-border-color);
+  --_default-border-width: var(--_secondary-border-width);
+  --_active-color: var(--_secondary-active-color);
+  --_active-background-color: var(--_secondary-active-background-color);
+  --_active-border-color: var(--_secondary-active-border-color);
+  --_active-border-width: var(--_secondary-active-border-width);
+  --_focus-color: var(--_secondary-focus-color);
+  --_focus-background-color: var(--_secondary-focus-background-color);
+  --_focus-border-color: var(--_secondary-focus-border-color);
+  --_hover-color: var(--_secondary-hover-color);
+  --_hover-background-color: var(--_secondary-hover-background-color);
+  --_hover-border-color: var(--_secondary-hover-border-color);
+  --_hover-border-width: var(--_secondary-hover-border-width);
+}
+
+:host([variant="secondary" i][danger]) button {
+  --_default-color: var(--_danger-background-color);
+  --_default-background-color: transparent;
+  --_default-border-color: var(--_danger-background-color);
+  --_active-color: var(--_danger-background-color);
+  --_active-background-color: transparent;
+  --_active-border-color: var(--_danger-active-border-color);
+  --_focus-color: var(--_danger-background-color);
+  --_focus-background-color: transparent;
+  --_focus-border-color: var(--_danger-focus-border-color);
+  --_hover-color: var(--_danger-background-color);
+  --_hover-background-color: transparent;
+  --_hover-border-color: var(--_danger-hover-border-color);
+}
+
```

Comment:
For the `secondary danger` button in dark theme, `--_default-color` should be `--rh-color-red-200`. (Changed this for good contrast on all dark surface backgrounds and to be a bit more consistent with PF's buttons.)